src/modules/CreateDrops/NftDrop/Fields/NftNameInput.tsx

```diff
@@ -0,0 +1,23 @@
+import { Control, Controller } from 'react-hook-form';
+
+import { TextInput } from '@/common/components/TextInput';
+
+import { CreateNftDropFormFieldTypes } from '../CreateNftDropForm';
+
+interface Props {
+  control: Control<CreateNftDropFormFieldTypes, any>;
+}
+
+const FIELD_NAME = 'nftName';
+
+export const NftNameInput = ({ control }: Props) => {
+  return (
+    <Controller
+      control={control}
+      name={FIELD_NAME}
+      render={({ field, fieldState: { error } }) => (
+        <TextInput errorMessage={error?.message} label="Name" placeholder="Danny Daze" {...field} />
```

Comment:
Can we use our custom `FormControl` in `TextInput`? the error text are not showing